src/types/onyx/CsvExportLayout.ts

```diff
@@ -0,0 +1,9 @@
+import type * as OnyxCommon from './OnyxCommon';
+
+/** Information about account level in-app export templates */
+type CsvExportLayout = OnyxCommon.OnyxValueWithOfflineFeedback<{
```

Comment:
This is identical to IntegrationServerExportTemplate so we could generalize both into something like `ExportTemplate` instead 🤔